$NetBSD: patch-ag,v 1.3 1999/03/04 14:56:44 agc Exp $

Don't check ncurses lib - just use standard curses.
Install files into share hierarchy
Don't assume that we're using the install.sh script - comment out
	the attempt to prefix it with ../

--- configure.orig	Tue Mar  3 13:30:18 1998
+++ configure	Sun Oct  4 15:53:12 1998
@@ -1401,7 +1401,7 @@
 # Some GNU/Linux systems (e.g., SuSE 4.3, 1996) don't have curses, but
 # rather ncurses.  So we check for it.
 TERMLIBS=
-for termlib in ncurses curses termcap terminfo termlib ; do
+for termlib in curses termcap terminfo termlib ; do
    echo $ac_n "checking for tputs in -l${termlib}""... $ac_c" 1>&6
 echo "configure:1407: checking for tputs in -l${termlib}" >&5
 ac_lib_var=`echo ${termlib}'_'tputs | sed 'y%./+-%__p_%'`
@@ -3443,7 +3443,7 @@
 		 USE_INCLUDED_LIBINTL=yes
 		 CATOBJEXT=.cat
 		 INSTOBJEXT=.cat
-		 DATADIRNAME=lib
+		 DATADIRNAME=share
 		 INTLDEPS='$(top_builddir)/intl/libintl.a'
 		 INTLLIBS=$INTLDEPS
 		 LIBS=`echo $LIBS | sed -e 's/-lintl//'`
@@ -3990,10 +3990,13 @@
     top_srcdir="$ac_dots$ac_given_srcdir" ;;
   esac
 
+# do not assume that we're using the install.sh script...
+if false; then
   case "$ac_given_INSTALL" in
   [/$]*) INSTALL="$ac_given_INSTALL" ;;
   *) INSTALL="$ac_dots$ac_given_INSTALL" ;;
   esac
+fi
 
   echo creating "$ac_file"
   rm -f "$ac_file"
